tests/unit_tests/test_rectangle.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+from SyntheticDataset import *
+import unittest
+
+class RectangleTestCase(unittest.TestCase):
+
+    def setUp(self):
+        self.height = 8
+        self.x_initial = 0
+        self.y_initial = 0
+        self.width = 6
+        self.rectangle = Rectangle(self.x_initial, self.y_initial, self.width, self.height)
+
+    def test__repr__(self):
+        self.assertEquals("X: 0 Y: 0 Width: 6 Height: 8", self.rectangle.__repr__())
+
+    def test_get_Height(self):
```

Comment:
Fix method naming to be lowercase please

Reply:
done